lib/graphql/analysis/analyze_query.rb

```diff
@@ -1,6 +1,9 @@
 module GraphQL
   module Analysis
     module_function
+
+    attr_reader :errors
```

Comment:
:tada: surprise, this is a module 😛 I think instance attributes on it will be shared across threads, resulting in :boom:. Currently, the local variable `analyzers_and_values` contains all state.